components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -0,0 +1,158 @@
+ */
++ (nonnull instancetype)actionSheetControllerWithTitle:(nullable NSString *)title
+                                               message:(nullable NSString *)message;
+
+- (nonnull instancetype)initWithNibName:(NSString *)nibNameOrNil
+                                 bundle:(NSBundle *)nibBundleOrNil;
+
+- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder;
+
+/**
+ Adds an action to the action sheet.
+
+ Actions are the possible reactions of the user to the presented alert. Actions are added as a
+ list item in the list of the action sheet.
+ Action buttons will be laid out from right to top to bottom depending on the the order they
+ were added, first being top last being bottom.
+
+ @param action Will be added to the end of MDCActionSheetController.actions.
+ */
+- (void)addAction:(nonnull MDCActionSheetAction *)action;
+
+/**
+ The actions that the user can take in response to the action sheet.
+
+ The order of the actions in the array matches the order in which they were added to the action sheet.
+ */
+@property (nonatomic, nonnull, readonly) NSArray<MDCActionSheetAction *> *actions;
+
+/**
+ The title of the action sheet controller.
+ */
+@property (nonatomic, nullable, copy) NSString *title;
+
+/**
+ The message of the action sheet controller.
+ */
+@property (nonatomic, nullable, copy) NSString *message;
+
+/*
+ Indicates whether the button should automatically update its font when the device’s
+ UIContentSizeCategory is changed.
+
+ This property is modeled after the adjustsFontForContentSizeCategory property in the
+ UIConnectSizeCategoryAdjusting protocol added by Apple in iOS 10.0.
+
+ If set to YES, this button will base its text font on MDCFontTextStyleButton.
+
+ Defaults value is NO.
+ */
+@property(nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
+BOOL mdc_adjustsFontForContentSizeCategory;
+
+@end
+
+/**
+ MDCActionHandler is a block that will be invoked when the action is selected.
+ */
+typedef void (^MDCActionSheetHandler)(MDCActionSheetAction *_Nonnull action);
+
+/**
+  MDCActionSheetAction is passed to MDCActionSheetController to add a button to the action sheet.
+ */
+@interface MDCActionSheetAction : NSObject <NSCopying>
+
+
+/**
+  Action alerts control the list items that will be displayed on the bottom of an action
+ sheet controller.
+
+ @param title The title of the list item shown on the list
+ @param image The icon of the list item shown on the list
+ @param handler A block to execute when the user selects the action.
+ @return An initialized MDCActionSheetAction object.
+ */
++ (nonnull instancetype)actionWithTitle:(nonnull NSString *)title
+                                  image:(nonnull UIImage *)image
+                                handler:(__nullable MDCActionSheetHandler)handler;
+
+/** Action sheet actions must be created with actionWithTitle:image:handler: */
+- (nonnull instancetype)init NS_UNAVAILABLE;
+
+/**
+ Title of the cell shown on the action sheet.
```

Comment:
Remove any reference to `cell` in the public docs - this term leaks an implementation detail that could change in the future.

Reply:
Done